README.md

```diff
@@ -22,6 +22,9 @@ GH_CLIENT_ID=xyz123
 GH_CLIENT_SECRET=asdf123
 # The URL to redirect the user to once the authentication was successful
 REDIRECT_URL=https://google.com
+
+# An optional GH_HOST if you're using GitHub Enterprise; defaults to `github.com`
```

Comment:
nit: Let's change this to "Optional: Specify the GitHub host when using GitHub Enterprise"